pkg/operator2/configobservation/console/observe_consoleurl.go

```diff
@@ -0,0 +1,43 @@
+package infrastructure
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+
+	"github.com/openshift/cluster-authentication-operator/pkg/operator2/configobservation"
+)
+
+func ObserveConsoleURL(genericlisters configobserver.Listers, recorder events.Recorder, previouslyObservedConfig map[string]interface{}) (map[string]interface{}, []error) {
+	assetPublicURLPath := []string{"oauthConfig", "assetPublicURL"}
```

Comment:
thanks, so I think I am going to add oauth-server's config under `oauthServerConfig`.

have you tested it?

Reply:
I played with the previous PR which seemed to have worked fine to observable degree